test_unstructured_inference/test_api.py

```diff
@@ -8,6 +8,8 @@
 from unstructured_inference.inference.layout import DocumentLayout
 import unstructured_inference.models.detectron2 as detectron2
 
+import jsons
```

Comment:
Is this different from `json` in the standard library?

Reply:
Allows auto deserializing using the class code from json. I tried for checking if make some intelligent thing about images, but at the end was just convenient.
https://jsons.readthedocs.io/en/latest/